test/integration_test/common_test.go

```diff
@@ -126,6 +126,25 @@ func TestSnapshot(t *testing.T) {
 	t.Run("testWebhook", testWebhook)
 }
 
+func TestStorkCbt(t *testing.T) {
+	t.Run("deploymentTest", deploymentMigrationTest)
+	t.Run("deploymentMigrationReverseTest", deploymentMigrationReverseTest)
```

Comment:
Can we use failoverAndFailback test instead? It is a superset of the deploymentMigrationReverseTest

Reply:
Done.

Reply:
@adityadani added vendor changes as discussed.